File: app/src/main/java/com/bubelov/coins/api/rates/BitcoinAverageApi.java
package com.bubelov.coins.api.rates;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * @author Igor Bubelov
 */

public interface BitcoinAverageApi {
    @GET("ticker/global/USD")
    Call<BitcoinAverageTickerResponse> getUsdTicker();
}

File: app/src/main/java/com/bubelov/coins/api/rates/BitcoinAverageApi.kt
package com.bubelov.coins.api.rates

import com.google.gson.JsonObject
import retrofit2.Call
import retrofit2.http.GET

/**
 * @author Igor Bubelov
 */

interface BitcoinAverageApi {
    @GET("indices/global/ticker/short?crypto=BTC&fiat=USD")
    fun getUsdTicker(): Call<JsonObject>
}